Raymond-Osy/Politico
# #163310609 API endpoint for getting all political parties

#### What does this PR do?
It's an API endpoint for getting all parties

#### Description of Task to be completed?
- Implement feedback for welcome route and response style guide format
- Add parties route to handle an incoming request to this endpoint
- Add parties controller that contains logic for the response
- Add a dummy database to store our data structure

#### How should this be manually tested?
- clone this app to your local computer
- open your console/command line
- type in 'npm run dev'
- open your browser or your postman and on the url type in http://localhost:8000/api/v1/parties/
- a list of all the political parties should be displayed

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
